samples/msal-browser-samples/VanillaJSTestApp2.0/app/customizable-e2e-aad/test/browser.spec.ts

```diff
@@ -0,0 +1,307 @@
+import puppeteer from "puppeteer";
+
+import { Screenshot, createFolder, setupCredentials, enterCredentials, ONE_SECOND_IN_MS } from "../../../../../e2eTestUtils/TestUtils";
```

Comment:
We can. However, since `jest` transforms `ts` files into `js`, it requires maintaining `moduleNameMapper` for each innermost `jest.config.js`.  [tsconfig-paths](https://www.npmjs.com/package/tsconfig-paths) does not work with `jest` either. I would suggest sticking to relative paths for simplicity.